openlibrary/macros/ListCarousel.html

```diff
@@ -1,4 +1,4 @@
-$def with (list_key, title=None, sort='new', key='', limit=20, has_fulltext_only=True, url=None)
+$def with (list_key, title=None, sort='new', key='', limit=20, has_fulltext_only=True, url=None, grid_mode=False)
```

Comment:
Oh one annoying change, lets change this to `layout='grid'` or `layout='carousel'` ; we might have other layouts in the future (eg `layout='details') and this might make it a little easier to handle!

Reply:
Good idea, I was looking at some ideas to style lists if the librarian had the ability to add some text to each item so that would help provide a framework for that too.